### ISSUE TITLE
Create managment command to setup basic component structure

### ISSUE THREAD

**Issue by EmilStenstrom** [EmilStenstrom/django-components#249]
It would be nice with a management command that generates a component with all the supporting files, and a default directory structure. There was an attempt to do this at https://github.com/EmilStenstrom/django-components/pull/175 but unfortunately that PR is stale now. I think it might be easiest to start over.

**Comment by hanifbirgani** [comment]
The boilerplate content of `componenet-name.py` is straightforward. What should be the default content of other generated files? (js/css/html)

**Comment by EmilStenstrom** [comment]
Happy to hear that you are working on this! :) 

There's a discussion about a new default structure that ended up in a suggestion to change the default names of files to template.html, style.css, and index.js. 

I think it makes sense to keep the contents of the files small, but still demonstrate how you connect them. Suggestion:

template.html
```django
<div class="component-[[ insert component name ]]">
    Hello component!
</div>
```

style.css
```css
.component-[[ insert component name ]] {
    background: pink;
}
```

index.js
```js
window.addEventListener("load", (event) => {
  console.log("[[ insert component name ]] is fully loaded");
});
```

What do you think?

**Comment by hanifbirgani** [comment]
@EmilStenstrom I've just added #299 PR to address this issue.

**Comment by nerdoc** [comment]
I would maybe use cookiecutter for this, and define a customizable URL where to find that template. If no CC template is given, use a default

**Comment by EmilStenstrom** [comment]
Interesting idea! What advantages would that give compared to this version?

**Comment by nerdoc** [comment]
I wrote a similar Commando for GDAPS, a Django Plugin system, to create plugins Form templates. First I copied Django's startapp cmd, and found it very in flexible. Cookiecutter already has solved many problems, e.g. flexibly naming directories.

Templates with cc could live outside the main repo, and could get updates without adding a new version (which can be an advantage or disadvantage).

And most importantly, user could add own cc templates which could be used easily using e.g.

    ./manage.py start component my_component --URL github.com/foouser/footemplate

Sure, this could be solved locally, but online repos help to keep diversity, and sharing templates.
